internal/buildscripts/packaging/jmx-metric-gatherer-release.txt

```diff
@@ -0,0 +1 @@
+1.7.0-alpha
```

Comment:
This is quite old: https://central.sonatype.com/artifact/io.opentelemetry.contrib/opentelemetry-jmx-metrics/1.25.1-alpha/versions